README.md

```diff
@@ -1,4 +1,4 @@
-# draft-store-client
+# @hmcts/draft-store-client
```

Comment:
no need to change this

Reply:
bug in github or their caching, have tested locally and the commit changing this is there